src/components/register/Register.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { useState } from 'react'
+import { registerUser } from "../utils"
+
+const Register = () => {
+    const [username, setUsername] = useState()
+    const [email, setEmail] = useState()
+    const [password, setPassword] = useState()
+
+    const submitHandler = async (e) => {
+        e.preventDefault()
+        await registerUser(username, email, password)
+    }     
+
+    return (
+        <>
+            <h2>Register New Account</h2>
+            <form onSubmit={submitHandler}>
+                <label>Username:
+                    <input onChange={(e) => setUsername(e.target.value) } required />
+                </label>
+                <br/>
+                <label>Email:
```

Comment:
Email address is not required for registration within this week's project. (backend/user/register.txt).

I realise I made the same mistake in the designs on Figma, which didn't help.